tests/conftest.py

```diff
@@ -145,8 +144,7 @@ def _setup_database(_configure_planetmint):  # TODO Here is located setup databa
 
 @pytest.fixture
 def _bdb(_setup_database, _configure_planetmint):
-    print(f"BDB CALL")
-    from planetmint.backend import connect
+    print("BDB CALL")
```

Comment:
@ArpitShukla007 can you remove this print statement? I think it got placed there for debugging purposes